docs/DeveloperGuide.adoc

```diff
@@ -1128,6 +1146,215 @@ Use case resumes at step 1.
 +
 Use case resumes at step 1.
 
+[discrete]
+=== Use case: Creating a class
+
+*Pre-condition*
+
+Module code exists in data file.
+
+*MSS*
+
+1.  User enters command to create classroom.
+2.  Classroom is created for the module.
+3.  System displays message of successful creation of class.
+
+*Extensions*
+
+[none]
+* 1a. User entered invalid command.
++
+[none]
+** 1a1. System shows ‘invalid format’ error.
+
+Use case resumes at step 1.
```

Comment:
Please add a `+` above all your lines of `Use case resumes at step 1` so that it is inline with the extension

Reply:
Also, can you add `Use case ends.` after every MSS? You may refer to the AB4 `Delete Person` use case at the top of the section for the correct styling